app/domain/operations/families/verifications/dmf_determination/request_dmf_determination.rb

```diff
@@ -0,0 +1,144 @@
+            handle_dmf_error("Could not find Transmittable::Job with job_id #{job_id}")
+          end
+
+          def construct_request_values(family_hbx_id)
+            Success({
+                      key: @job&.key,
+                      title: @job&.title,
+                      description: @job&.description,
+                      correlation_id: family_hbx_id.to_s,
+                      started_at: @job&.started_at,
+                      publish_on: @job&.publish_on
+                    })
+          end
+
+          def build_and_create_request_transmission(values)
+            values[:key] = :dmf_determination_request
+            values[:event] = 'initial'
+            values[:state_key] = :initial
+            values[:job] = @job
+
+            create_request_transmission(values, @job)
+          end
+
+          def build_and_create_request_transaction(values, family)
+            values[:key] = :dmf_determination_request
+            values[:event] = 'initial'
+            values[:state_key] = :initial
+            values[:transmission] = @transmission
+            values[:subject] = family
+
+            create_request_transaction(values, @job)
+          end
+
+          def build_cv3_family_payload(family)
+            cv3_family = Operations::Transformers::FamilyTo::Cv3Family.new.call(family)
+
+            if cv3_family.success?
+              @transaction.json_payload = cv3_family.value!.to_h
+              @transaction.save
+
+              payload = { family_hash: @transaction.json_payload, job_id: @job.job_id }
+              @transaction.json_payload ? Success(payload) : Failure("Unable to save transaction with payload")
+            else
+              message = "Unable to transform family into cv3_family"
+              add_errors(message, :build_and_validate_cv3_family, { job: @job, transmission: @transmission, transaction: @transaction })
+              status_result = update_status(message, :failed,
+                                            { job: @job, transmission: @transmission, transaction: @transaction })
+              return status_result if status_result.failure?
+              result
+            end
+          rescue StandardError
+            add_errors("Unable to save transaction with payload", :generate_transmittable_payload,
+                       { job: @job, transmission: @transmission, transaction: @transaction })
+            status_result = update_status("Unable to save transaction with payload", :failed,
+                                          { job: @job, transmission: @transmission, transaction: @transaction })
+            return status_result if status_result.failure?
+            result
+          end
+
+          def build_event(payload)
+            event('events.families.verifications.dmf_determination.requested', attributes: payload)
+          end
+
+          def publish(event)
+            hbx_id = @transaction.json_payload[:hbx_id]
+            event.publish
```

Comment:
We need to record verification history. For successful request and also failure. Or a person has a invalid ssn's.